content/en/stories/_index.md

```diff
@@ -184,6 +184,9 @@ aliases:
       {{< /company >}}
       {{< company name="Otto Group" image="/images/logos/ottogroup.png" article="https://github.com/customer-stories/otto-group" >}}
       {{< /company >}}
+      {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/en-us/story/1671022200964097988-connect-panasonic-discrete-manufacturing-azure-en-japan" author_title="Panasonic Connect Adopts Azure and GitHub to Transform to a Cloud-Native Development Culture" >}}
+      The Process of InnerSource is actually the mindset of Collaboration.
```

Comment:
```suggestion
```

I suspect this should be removed, as it is the same as for the company below?

Reply:
Yest definitely! Thank you!